File: Android/DaycareParent/DaycareParent/app/src/main/java/com/daycare/daycareparent/model/TeacherClassCheckInModel.java
package com.daycare.daycareparent.model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class TeacherClassCheckInModel {
    @SerializedName("totalPages")
    public Integer totalPages;
    @SerializedName("totalRows")
    public Integer totalRows;
    @SerializedName("pageSize")
    public Integer pageSize;
    @SerializedName("id")
    public Integer id;
    @SerializedName("isExist")
    public Boolean isExist;
    @SerializedName("saveId")
    public Integer saveId;
    @SerializedName("statusCode")
    public Integer statusCode;
    @SerializedName("isSuccess")
    public Boolean isSuccess;
    @SerializedName("message")
    public String message;
    @SerializedName("data")
    public List<Datum> data = null;

    public class Datum {

        @SerializedName("value")
        public Integer value;
        @SerializedName("label")
        public String label;
        @SerializedName("isActive")
        public Boolean isActive;
    }
}

File: Android/DayCareTeacher/DayCareTeacher/app/src/main/java/com/daycare/daycareteacher/model/TeacherClassCheckInModel.kt
package com.daycare.daycareteacher.model

import com.google.gson.annotations.SerializedName

class TeacherClassCheckInModel {
    @SerializedName("totalPages")
    var totalPages: Int? = null
    @SerializedName("totalRows")
    var totalRows: Int? = null
    @SerializedName("pageSize")
    var pageSize: Int? = null
    @SerializedName("id")
    var id: Int? = null
    @SerializedName("isExist")
    var isExist: Boolean? = null
    @SerializedName("saveId")
    var saveId: Int? = null
    @SerializedName("statusCode")
    var statusCode: Int? = null
    @SerializedName("isSuccess")
    var isSuccess: Boolean? = null
    @SerializedName("message")
    var message: String? = null
    @SerializedName("data")
    var data: List<Datum>? = null

    inner class Datum {

        @SerializedName("value")
        var value: Int? = null
        @SerializedName("label")
        var label: String? = null
        @SerializedName("isActive")
        var isActive: Boolean? = null
    }
}
